pandas-stubs/core/frame.pyi

```diff
@@ -1906,22 +1906,22 @@ class DataFrame(NDFrame, OpsMixin):
         self,
         axis: AxisType | None = ...,
         skipna: _bool | None = ...,
+        level: None = ...,
         numeric_only: _bool | None = ...,
         min_count: int = ...,
-        *,
-        level: Level,
```

Comment:
Is that true for a MultiIndex? If not, it might be good to mention that the stubs focus on getting the most common usage patterns rights (as pandas can not completely be typed because of the current type system/pandas API), maybe in the README (I think numpy has a "limitation"/"difference from implementation" section on their type page).

Reply:
If you specify a `level`, then you get a different result than when you don't specify it.

I did write this here: https://github.com/pandas-dev/pandas-stubs/blob/main/docs/philosophy.md

> The goal of the pandas-stubs project is to provide type stubs for the public API that represent the recommended ways of using pandas.

Reply:
When running this:
```py
type(pd.DataFrame({"a": [1], "b": [2], "c": [3]}).set_index(["a", "b"]).sum(level="a"))  # <class 'pandas.core.frame.DataFrame'>
FutureWarning: Using the level keyword in DataFrame and Series aggregations is deprecated and will be removed in a future version. Use groupby instead. df.sum(level=1) should use df.groupby(level=1).sum().
```

I wouldn't mind omitting the level argument in sum.

Reply:
Can you create an issue and we'll do a separate PR for that?